.npmrc

```diff
@@ -0,0 +1,3 @@
+auto-install-peers=false
+strict-peer-dependencies=false
+dedupe-peer-dependents=false
```

Comment:
why do we need all this

Reply:
To do what it says. Is clear no?

I'm joking, I have no idea. I was trying